groovy/inductor_graph.groovy

```diff
@@ -0,0 +1,167 @@
+NODE_LABEL = 'mlp-validate-icx24-ubuntu'
+if ('NODE_LABEL' in params) {
+    echo "NODE_LABEL in params"
+    if (params.NODE_LABEL != '') {
+        NODE_LABEL = params.NODE_LABEL
+    }
+}
+
+target_tag = ''
+if ('target_tag' in params) {
+    echo "target_tag in params"
+    if (params.target_tag != '') {
+        target_tag = params.target_tag
+    }
+}
+echo "target_tag: $target_tag"
+
+reference_tag = ''
+if ('reference_tag' in params) {
+    echo "reference_tag in params"
+    if (params.reference_tag != '') {
+        reference_tag = params.reference_tag
+    }
+}
+echo "reference_tag: $reference_tag"
+
+reference = ''
+if ('reference' in params) {
+    echo "reference in params"
+    if (params.reference != '') {
+        reference = params.reference
+    }
+}
+echo "reference: $reference"
+
+channels = ''

```

Comment:
We'd better to give them a default value, e.g. `channels = 'first'` ,  also for `suite` / `model` / `DT` / `SHAPE`